Schedule layout, CZ lang

PetrM97 opened the pull request:
I have added a new layout for scheduling some repeated task (for example cleaning a shared space) and a new style bw_transparent, that can apply color to every day cell.
I have also added a Czech language.
I have changed the item_span(n, k) function that I use in my schedule layout (now it should span from n-th cell to (n+k)-th cell).

geotz commented:
Hi, thanks for your interest in Callirhoe!

I am afraid I cannot merge this directly:

The CZ language pack is OK. You can send me a separate PR for that or I can merge the translations when I get some time. You can also put your name if you want (see DE and TR for an example).

The schedule is a cool idea, however it needs refactoring for an official release:
* The `classic` layout is broken! Didn't have time to debug... maybe due to `item_span` change?
* The duplicated code should be as little as possible. In fact, the schedule should render 
a classic layout in a sub-rectangle. Then it can render the title and the legend (title and legend could be toggleable). The minor visible modifications could be overridden virtual methods that perform some decorations to the original rendering.
* There should be some interface to declare/configure the users and colors, either command-line or maybe use the holidays file.
* In fact I wasn't able to run the code! I had to patch it. I got problems invoking the `S.dom` attribute constructor with keyword arguments... which python version are you using? (I'm not used to calling constructors like that).

PetrM97 commented:
Thank you for your feedback.

- I changed the `item_span` function as the comment in the function states `get union of I{k} consecutive items, starting at position I{n}` (https://github.com/geotz/callirhoe/blob/f410685e4277f98934be21fd7a04f92086c15382/lib/geom.py#L185). However the function gets union of `I{n} consecutive items, starting at position I{k}`, so I got confused when I first saw it. I just did not realize, that I broke the classic layout by changing this function.
- I just copied the `_base` and `classic` layouts and changed just a few lines, so I agree that the code is too much duplicated. When I wrote the code I just 'hacked' a few lines of code without considering the maintainability. 
- I will consider adding command line arguments for toggling header, legend, names and colors (maybe in the future).
- I was using Python 3.8. I temporarily changed the header in `callirhoe.py` to use this version and maybe wrongly assumed that it will work in Python 3.7.

I consider this pull request closed, as I understand your comments about it.
